packages/datadog-plugin-opensearch/test/index.spec.js

```diff
@@ -244,7 +244,8 @@ describe('Plugin', () => {
 
         withPeerService(
           () => tracer,
-          (done) => client.search({
```

Comment:
I'm surprised the unused `done` didn't throw a linter error before

Reply:
Indeed. I tried finding where in eslint this would trip up, `no-unused-vars`'s doc seems to suggest it should, but it doesn't...